doc/modules/model_evaluation.rst

```diff
@@ -1536,16 +1530,18 @@ Here is a small example of usage of this function:::
     >>> brier_score_loss(y_true, y_prob > 0.5)
     0.0
 
-The Brier score can be used to assess how well a classifier is calibrated
-however, a lower Brier score does not always mean a better calibration. This is
-because the Brier score can be decomposed as the sum of calibration loss and
-refinement loss [Bella2012]_. Calibration loss is defined as the mean squared
-deviation from empirical probabilities derived from the slope of ROC segments.
-Refinement loss can be defined as the expected optimal loss as measured by the
-area under the optimal cost curve. Refinement loss can change independently
-from calibration loss, thus a lower Brier score does not necessarily mean a
-better calibrated model. "Only when refinement loss remains the same does a
-lower Brier score always mean better calibration" [Bella2012]_, [Flach2008]_.
+The Brier score can be used to assess how well a classifier is calibrated.
+However, a lower Brier score loss does not always mean a better calibration.
```

Comment:
If we are using "Brier score loss" here, do we do it everywhere in the paragraph?

Reply:
I took the liberty to specify "Brier score loss" in 2 other places as well.